src/Panel/Panel.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "../less/index";
+
+.Panel--container {
```

Comment:
I think he means that if `container` is a component, this should be `.Panel--Container` to denote that `Container` is a component; but if it's not I don't think you need to capitalize it, correct me if I'm wrong @CarlNelson